restapi/src/main/java/io/stargate/web/docsapi/resources/DocumentResourceV2.java

```diff
@@ -712,15 +712,14 @@ public Response getDocPath(
           List<FilterCondition> filters = new ArrayList<>();
           List<String> selectionList = new ArrayList<>();
 
+          if (fields != null) {
+            JsonNode fieldsJson = mapper.readTree(fields);
+            selectionList = documentService.convertToSelectionList(fieldsJson);
```

Comment:
This method has all the bugs that I solved in #1047.. Your fields will not be converted to proper array, glob or path segment values.

Reply:
Rebased to reuse `DocApiUtils`. Please re-review.